src/forms/common/CommonApplicationForm.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import { useEffect } from 'react';
+
+import { Form } from 'lattice-fabricate';
+import { ReduxUtils } from 'lattice-utils';
+
+import { schema, uiSchema } from './schemas';
+
+import { SUBMIT_ACCESS_REQUEST, submitAccessRequest } from '../../containers/access/src/actions';
+import { useDispatch, useSelector } from '../../core/redux';
+import { resetRequestState } from '../../core/redux/actions';
+import { ACCESS, REQUEST_STATE } from '../../core/redux/constants';
+import { goToRoot } from '../../core/router/actions';
+
+const { isPending, isSuccess } = ReduxUtils;
+
+const CommonApplicationForm = () => {
+  const dispatch = useDispatch();
+  const requestState = useSelector((s) => s.getIn([ACCESS, SUBMIT_ACCESS_REQUEST, REQUEST_STATE]));
```

Comment:
No change required, just a thought. Unfortunately, `useRequestState` doesn't work with the custom `useSelector` in modules. It would be nice to fix that 🙂

Reply:
This is because the module uses its own store and thus requires its own context.